mkdocs/themes/mkdocs/locales/fr/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,100 @@
+# French translations for MkDocs.
+# Copyright (C) 2021 MkDocs
+# This file is distributed under the same license as the MkDocs project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
```

Comment:
Why are these values all placeholders?

Reply:
I'm guessing that po file was generated before the pot file was properly created. But, yes, all of the po files need to be updated to make use of the proper meta-data from the pot file. To be clear, I have not confirmed that the metadata in the pot file is all correct. That should probably happen first.

Reply:
Indeed @waylan 

But the thing is, `update_catalog` does NOT update those values afterwards so I'm unsure how much we want this kind of metadata to be up to date or even used TBH... I personally find them useless which is why I proposed to strip them at first..

What's your expectations there?

For example, what should I put in:

```
"Report-Msgid-Bugs-To: github@mkdocs.org\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
```

?

Reply:
Friendly bump @waylan

Reply:
Regarding the suggestion to remove the data fields entirely; I'm undecided. For the most part, the same info can be obtained from the git commit, so their value is questionable. That said, by having the info directly in the file makes it easy to find. If the fields are completely removed from the pot file, will creating a new po file also exclude them? If they get added back into any new po files (created from the appropriate command), then we should definitely leave them in. The rest of this comment assumes we are keeping the fields.

Looking at the items with placeholders, they could all be considered to be local specific (the data could be different for each local). So I assume that they don't carry over from the pot file, but need to be completed by the translator in each po file. 

Do the command-line commands accept values for these? If so, perhaps we could add some additional code to our wrappers which retrieved info from the git user (for example) and passed that in automatically. If not, then we will need to document the need for translators to manually edit these fields before submitting a new or updated translation. Failure to provide this info would be a blocker for any PR with new and/or updated po files.

Out of respect for other's privacy, we might also want to document that email addresses are not strictly required. However, some way of uniquely identifying a translator should be included. I would be okay with an `@mention` using a GitHub username. For example, I might use this:

```
"Last-Translator: Waylan Limberg @waylan\n"
```

That said, if a translator prefers to use their email address, I'm okay with that as well. However, I don't think it makes sense to allow usernames for other sites not affiliated with the MkDocs project.

@ultrabug obviously, you will need to fill in these values for the existing po files as you are the translator.

Regarding the actual data to use...

`Project-Id-Version` should be passed in from `setup.py` if possible. If that's not possible, then we need to document the need for translators to manually update it. Right now, the pot file has `MkDocs 1.2.dev1` which is the correct 'current' version. However, when we do a release, the current version will be `MkDocs 1.2` at that time.  I realize that if more changes are made to the template after the translation, then the release version may no longer be correct for the po file. That said, the dev version always stays `dev1` so there is no real value in including that. Maybe if we incremented the dev version is would make more sense. Until then, I think the assumed future release version is sufficient here (`MkDocs 1.2` at this time).

Set `Report-Msgid-Bugs-To` to  `https://github.com/mkdocs/mkdocs/issues` by default. I know that's not an email address, but that is where bugs should be reported. If the translator for a specific local wants to use something different, then that can be discussed if/when it comes up.

For `Language-Team` we can remove it (or leave it blank).  My thinking is that if a team of translators does form, they can add that back in if they choose. The translator should be adding his/her own info in the `Last-Translator:` field, so there is contact info available already. Given the small number of translatable terms in our templates, I doubt we will have many "teams", if any.

Of course, `Last-Translator` should always be filled in/updated by the translator.  Same goes for `PO-Revision-Date` (its a shame that doesn't auto-update).

Finally, `FIRST AUTHOR` in the comments at the top of each file could possibly be removed. I'm assuming it is intended to be part of the copyright notice. If a translator wants to include it, they can, but I don't think there is any value in requiring it. Of course, if a translator includes it, any future updates should not remove it. However, if the first translator does not feel the need to include it, then the placeholder should not be left in the po file.